.circleci/config.yml

```diff
@@ -0,0 +1,66 @@
+# Note: all paths are relative to the default workspace ~/project
+version: 2.1
+executors:
+  node-11:
+    docker:
+      - image: circleci/node:10
+jobs:
+  install-dependencies:
+    executor: node-11
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            # if lock file changes, we still use increasingly general patterns to restore cache
+            # the install afterwards will only install packages with different versions
+            - yarn-cache-{{ .Branch }}-{{ checksum "yarn.lock" }}
+            - yarn-cache-{{ .Branch }}-
+      - run:
+          name: Install dependencies
+          command: |
+            yarn --version
+            yarn install --frozen-lockfile
+            cd example
+            yarn install --frozen-lockfile
```

Comment:
I think install-dependencies should only include the core library, example can be installed in `unit-tests-example` step, what do you think?

Reply:
:+1: